src/server/src/main/java/io/cassandrareaper/ReaperApplication.java

```diff
@@ -253,6 +254,11 @@ public void run(ReaperApplicationConfiguration config, Environment environment)
     initializeJmxSeedsForAllClusters();
     LOG.info("resuming pending repair runs");
 
+    Preconditions.checkState(
+        !(!(context.storage instanceof IDistributedStorage)
+            && context.config.getDatacenterAvailability() == DatacenterAvailability.EACH),
```

Comment:
`context.storage instanceof IDistributedStorage || DatacenterAvailability.EACH != context.config.getDatacenterAvailability()`
would be a lot easier to read, imho.

Reply:
indeed :)